tests/e2e/specs/renders-omr-assessment.js

```diff
@@ -331,7 +331,7 @@ describe("Player for OMR quizzes", () => {
           cy.get('[data-test="questionSetInstruction-1"').should("exist");
           cy.get('[data-test="questionSetInstruction-1"').should(
             "have.text",
-            "Question Set 1 Description\nYou may attempt only up to 3 questions in this section."
+            "Question Set 1 DescriptionYou may attempt only up to 3 questions in this section."
```

Comment:
Shouldn't there be a `:` or something between `Description` and `You`?

Reply:
I removed `\n` and used <br> in instruction component. 
`have.text` was not capturing the `<br>` between "Description" and subsequent text